### ISSUE TITLE
Deep Learning 

### ISSUE THREAD

**Issue by BVishal-Geek** [BVishal-Geek/Year-2024-Progress#2]
Started deep learning from Neural Networks Course of Andrew NG

**Comment by BVishal-Geek** [comment]
Completed Gradient Descent. 

J(w,b) = 

1. w - learning_rate * (d J(w,b)/dw)
2. b - learning_rate * (d J(w,b)/db)

**Comment by BVishal-Geek** [comment]
Learned activation functions. 
1. Sigmoid
2. Tanh
3. ReLU
4. Leaky ReLU
5. Parametric ReLU
6. SWISH
7. Softplus
8. Softmax
